spec/features/select_2_spec.rb

```diff
@@ -84,6 +84,28 @@ def create_items
       end
     end
 
+    context "with non empty collection includes options" do
+      before do
+        register_form(Invoice) do |f|
+          f.input :number,
+                  as: :tags,
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.